### ISSUE TITLE
Unwanted "Figure" titles render in Jupyter Notebook interactive maps

### ISSUE THREAD

**Issue by Joshdpaul** [raphaelquast/EOmaps#195]
I just started using EOmaps in Jupyter Lab and I am running into a small issue. 

When rendering the interactive map, it seems to render multiple times (this was mentioned in another issue, possibly related to Jupyter Lab backend) but in my case this adds a default title to each rendering ("Figure", "Figure 1", "Figure 2", etc). Once the figure is done rendering, I am stuck with "Figure #" at the top. I have tried accessing the figure title through `m.f.suptitle()` but had no luck with that approach. Adding a title via `m.add_title()` works to add additional text, but does not remove the default "Figure #" title. Plotting via `m.snapshot()' does not show the unwanted title, but of course I lose the interactive functions of the map.

Expected behavior is that the standard rendering will not include any default figure titles. Below is a quick code example along with some screenshots to show what is being rendered in Jupyter Lab.

My system info:
- OS = linux
- python = 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0]
- EOmaps = 7.3.1
- I am actually using VSCode on MacOS Ventura 13.5.2, with a remote ssh connection to a system with the specs above ^^^

```
%matplotlib widget    
my_map = Maps(crs=4326, layer="base", figsize=(8, 6))
my_map.set_extent((-140.5, -129.5, 52.5, 60.5))  
my_map.add_feature.preset("coastline", "ocean", "land", "lakes", "rivers_lake_centerlines", scale=10)
my_map.add_wms.ESRI_ArcGIS.SERVICES.World_Shaded_Relief.add_layer.xyz_layer(layer='base')
my_map.add_title("SOUTHEAST ALASKA & VICINITY")

```
![image](https://github.com/raphaelquast/EOmaps/assets/99696041/93181898-7185-42a6-8fa5-06ad7812711b)

`my_map.snapshot()
`

![image](https://github.com/raphaelquast/EOmaps/assets/99696041/449be700-ed48-4fe9-8180-6934a366b5c9)

**Comment by raphaelquast** [comment]
@Joshdpaul good to hear :-)  
By the way... I've decided that this will become the new default starting with v7.3.2 (released soon) since the additional title is really annoying and I can't think of a situation where anyone wants to see it by default when working with EOmaps (e.g. it will be opt-in instead of opt-out)

v7.3.2 will also bring some fixes and improvements for inline-plotting in jupyter notebooks!
